File: smscentral.js
var request     = require('request'),
    url         = 'https://my.smscentral.com.au/wrapper/sms',
    username    = 'username',
    password    = 'password',
    action      = 'send',
    originator  = 'shared',
    recipient   = '0434390783',
    messagetext = 'Hello from SMSCentral!';


request.post(url, {form:{
            'USERNAME'  : username,
            'PASSWORD'  : password,
            'ACTION'    : action,
            'ORIGINATOR': originator,
            'RECIPIENT' : recipient,
            'MESSAGE_TEXT' : messagetext
        }},function(e, r, body){
            console.log(body);
        });

File: smscentral.py
import httplib, urllib

gateway = "my.smscentral.com.au"
requestpath = "/wrapper/sms"
username = "username"
password = "password"
recipient = "04xxxxxxxx"
messagetext = "Hello from SMSCentral"

params = urllib.urlencode({
        "USERNAME" : username,
        "PASSWORD" : password,
        "ACTION" : "send",
        "ORIGINATOR" : "shared",
        "RECIPIENT" : recipient,
        "MESSAGE_TEXT" : messagetext
})

headers = {"Content-type": "application/x-www-form-urlencoded",
           "Accept": "text/plain"}

conn = httplib.HTTPConnection(gateway)
conn.request("POST", requestpath, params, headers)

response = conn.getresponse()
data = response.read()
print data

conn.close()
